lib/cloudkeeper/aws/cloud.rb

```diff
@@ -29,6 +29,7 @@ def initialize(s3service: nil, ec2service: nil)
       # @yield [write_stream] output stream
       # @raise [Cloudkeeper::Aws::Errors::BackendError] if file already exists
       def upload_data(file_name, &block)
+        logger.debug { "Uploading file: #{file_name}" }
```

Comment:
Please also add to which bucket is image being uploaded. The same go for the `delete_data` method.